frontend/src/lib/DeltaParser.ts

```diff
@@ -101,10 +101,16 @@ function handleNewElementMessage(
 ): ReportElement {
   MetricsManager.current.incrementDeltaCounter(container)
   MetricsManager.current.incrementDeltaCounter(element.get("type"))
+
+  // Track component instance name.
+  if (element.get("type") === "componentInstance") {
+    const componentName = element.getIn(["componentInstance", "componentName"])
+    MetricsManager.current.incrementDeltaCounter(componentName)
+  }
```

Comment:
I think we probably want to do something like `"CustomComponent:${componentName}"`, but @tvst is the analytics master so let's verify with him first.

Whichever approach we choose, I think we should have `MetricsManager` know about custom components and send the appropriate data along , to make the expected call semantics clear. Maybe:
- `MetricsManager.incrementCustomComponentCounter(componentName)`
- or `MetricsManager.incrementDeltaCounter(deltaName: string, customComponentName: string | undefined = undefined)`

In other words, I don't think we should ever be passing a string like `"CustomComponent:${componentName}"` directly to `MetricsManager.incrementDeltaCounter` - if that's the strategy we want to use, then MetricsManager itself should implement it.